tensorflow/tools/test/run_and_gather_logs.py

```diff
@@ -40,10 +40,11 @@
 # pylint: enable=g-bad-import-order
 # pylint: enable=unused-import
 
-from google.protobuf import text_format
 from tensorflow.core.util import test_log_pb2
 from tensorflow.tools.test import run_and_gather_logs_lib
 
+from google.protobuf import text_format
```

Comment:
This change is necessary: tensorflow is considered a third-party library, which goes before google libraries.